packages/field-base/src/aria-label-controller.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Since the controllers are meant for internal use only, I'd imagine the ts definitions don't bring any extra value

Reply:
Let's keep typings so we could e.g. use controllers in a Lit + TS project for testing.